### PR TITLE
Fix grammar in nodes documentation

### PR DESCRIPTION
**Pull request by axsuul**
Just a simple grammar fix

### CODE REVIEW

**Comment by axsuul** [comment]
Signed CLA

**Comment by axsuul** [comment]
@Arhell All set I believe

**Comment by Arhell** [comment]
@axsuul Thank!
/assign @sftim

File: content/en/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The [components](/docs/concepts/overview/components/#node-components) on a node
 There are two main ways to have Nodes added to the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}:
 
 1. The kubelet on a node self-registers to the control plane
-2. You, or another human user, manually add a Node object
+2. You, or another human user, manually adds a Node object
```
Comment (sftim) [inline content/en/docs/concepts/architecture/nodes.md:35]: @axsuul I'm afrad that the original grammar is correct and this suggested replacement is not idiomatic English.

“You adds a Node object” does not read correctly; adding the “or” phrase does not change the required conjugation of _add_.

**Comment by sftim** [comment]
/close

In https://github.com/kubernetes/website/pull/22377#pullrequestreview-442972975 I explained that the original wording is idiomatic English.

**Pull request closed without merging**